addon/components/g-map-infowindow.js

```diff
@@ -43,10 +44,30 @@ const GMapInfowindowComponent = Ember.Component.extend({
     }
   },
 
+  optionsChanged: observer('disableAutoPan', 'maxWidth', 'pixelOffset', function() {
+    run.once(this, 'setOptions');
+  }),
+
+  setOptions() {
+    const infowindow = this.get('infowindow');
+    const options = ['disableAutoPan', 'maxWidth', 'pixelOffset'];
+    const infoWindowOptions = {};
+
+    for (let i = 0; i < options.length; i++) {
+      const value = this.get(options[i]);
+      if (isPresent(value)) {
+        infoWindowOptions[options[i]] = value;
+      }
+    }
+
+    if (isPresent(infowindow) && Object.keys(infoWindowOptions).length !== 0) {
+      infowindow.setOptions(infoWindowOptions);
+    }
+  },
+
   buildInfowindow() {
     const infowindow = new google.maps.InfoWindow({
-      content: this.get('element'),
-      disableAutoPan: true
```

Comment:
The reason for this was that my app had some conflicts with auto-panning to info windows against fitBounds panning, so I've made this quick restriction with intention to refactor it to full options setup later (which you've done here).